Packs/CommonTypes/IndicatorFields/indicatorfield-Resource_Level.json

```diff
@@ -1,52 +1,51 @@
 {
-    "closeForm": false,
-    "cliName": "campaign",
```

Comment:
@gal-berger do you understand what happened with this files?

Reply:
yes, please refresh and it will be fixed.